### PR TITLE
Added featurize datetime function

### PR DESCRIPTION
**Pull request by Justus-coded**

### CODE REVIEW

**Comment by AdesholaAfolabi** [comment]
Looks good to me. Will allow @Sensei-akin @ganiyuolalekan to have a look before merging.

**Review by Sensei-akin — APPROVED** [review]

File: slik_wrangler/preprocessing.py [hunk 1]
```diff
@@ -546,7 +546,7 @@ def get_attributes(data=None,target_column=None):
         cat_attributes.remove(target_column)
     else:
         pass
-    return num_attributes, cat_attributes
+    return {"Numerical Features": num_attributes, "Categorical Features": cat_attributes}
```
Comment (Sensei-akin) [inline slik_wrangler/preprocessing.py:549]: @Justus-coded Why did you make it a dictionary? Also the get_attributes fn is used in a couple of places and can cause some functions to break. cc @AdesholaAfolabi

File: slik_wrangler_app/pages/streamlit.py [hunk 2]
```diff
@@ -275,6 +277,14 @@ def main():
                     with st_stdout("info"):
                         transformed_df = pp.featurize_datetime(transformed_df,choice,feat_choice, drop)
 
+            if get_attr_fn:
+                #choice = st.sidebar.selectbox('select target column', transformed_df.columns.tolist())
+                col_list_ap = col_list.append(None)
+                index = len(col_list) -1
+                choice = st.sidebar.selectbox('exclude target column', col_list,key='get_attr',help="""
```
Comment (Sensei-akin) [inline slik_wrangler_app/pages/streamlit.py:284]: @Justus-coded why exclude target column. Can be select target column instead. cc @AdesholaAfolabi

Comment (Justus-coded) [reply slik_wrangler_app/pages/streamlit.py:284]: Wanted to use something more descriptive. "Select target column" doesn't let the user know that the target column will be excluded from the features.

File: slik_wrangler/preprocessing.py (hunk 1 above)
Comment (Justus-coded) [reply slik_wrangler/preprocessing.py:549]: My bad, didn't think about how my changes could affect the other places the function was used. 

The get_attr func returns both lists for num and cat features. The output from streamlit wasn't descriptive, that's why i used a dictionary.

File: slik_wrangler_app/pages/streamlit.py (hunk 2 above)
Comment (Sensei-akin) [reply slik_wrangler_app/pages/streamlit.py:284]: Yeah I saw. Can you revert to the previous return statement and handle the appearance on the streamlit script. You can return both objects and store in a dictionary instead

File: slik_wrangler/preprocessing.py (hunk 1 above)
Comment (Sensei-akin) [reply slik_wrangler/preprocessing.py:549]: Yeah I saw. Can you revert to the previous return statement and handle the appearance on the streamlit script. You can return both objects and store in a dictionary instead. This should help right?

Comment (Justus-coded) [reply slik_wrangler/preprocessing.py:549]: Yes, it should. 

Thank you.

File: slik_wrangler_app/pages/streamlit.py (hunk 2 above)
Comment (Justus-coded) [reply slik_wrangler_app/pages/streamlit.py:284]: Alright

Comment (AdesholaAfolabi) [reply slik_wrangler_app/pages/streamlit.py:284]: @Sensei-akin @Justus-coded I see the PR has been merged. Were the changes fixed before merging the PR?

**Pull request merged**